packages/tutorial/src/part4.tsx

```diff
@@ -0,0 +1,37 @@
+import fetch from 'node-fetch';
+import TurndownService from 'turndown';
+import * as LLMx from 'ai-jsx';
+import { DocsQA, ScoredChunk, LocalCorpus, makeChunker, staticLoader } from 'ai-jsx/batteries/docs';
+import { showInspector } from 'ai-jsx/core/inspector';
+
+const URL = 'https://en.wikipedia.org/wiki/2005_Azores_subtropical_storm';
+
+const html = await fetch(URL).then((response) => response.text());
+const turndownService = new TurndownService();
+const markdown = turndownService.turndown(html);
+const docs = [
+  {
+    pageContent: [markdown],
+    name: 'Wikipedia Article about 2005 Azores subtropical storm',
+  },
+];
+const corpus = new LocalCorpus(staticLoader(docs), makeChunker(300, 100));
+await corpus.startLoading();
+
+function ShowDoc({ doc }: { doc: ScoredChunk }) {
+  return doc.chunk.content;
+}
+
+function App() {
+  return (
+    <>
+      <DocsQA question="Summarize this article in a few sentences." corpus={corpus} docComponent={ShowDoc} />
+      {'\n\n'}
+      <DocsQA question="Which dates did the storm occur?" corpus={corpus} docComponent={ShowDoc} />
+      {'\n\n'}
+      <DocsQA question="Where were the strongest winds reported?" corpus={corpus} docComponent={ShowDoc} />
+    </>
```

Comment:
Maybe this provides a slightly nicer output?

```suggestion
function AskAndAnswer({children: question}: {children: Node}) {
    return <>
        Q: {question}
        {'\n\n'}
        A: <DocsQA question={question} corpus={corpus} docComponent={ShowDoc} />
    <>;
}


    <>
      <AskAndAnswer>Summarize this article in a few sentences.</AskAndAnswer>
      {'\n\n'}
      <AskAndAnswer>Which dates did the storm occur?</AskAndAnswer>
      {'\n\n'}
      <AskAndAnswer>Where were the strongest winds reported?</AskAndAnswer>
    </>
```

Reply:
It does, but it's more code than I want to present here. My goal is to keep things super simple in these tutorials.